presto-docs/src/main/sphinx/admin/resource-groups.rst

```diff
@@ -9,287 +9,452 @@ Except for the limit on queued queries, when a resource group runs out of a reso
+  exceeds the limit of 10, the user with the fewest running queries gets the next concurrency slot. This policy
   results in fairness when under contention.
 
 * All remaining queries are placed into a per-user group under ``global.adhoc.other`` that behaves similarly.
 
+File Resource Group Manager
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. literalinclude:: resources-groups-example.json
+    :language: json
+
+Database Resource Group Manager
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This example is for a MySQL database.
+
+.. literalinclude:: resource-groups-example1.sql
+    :language: sql
+
+.. code-block:: sql
+
+    -- global properties
+
+    -- get ID of 'other' group
+    SELECT resource_group_id FROM resource_groups WHERE name = 'other';  -- 4
+
+    -- create '${USER}' group with 'other' as parent.
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, environment, parent) VALUES ('${USER}', '10%', 1, 100, 'test_environment', 4);
+
+    -- create 'bi-${toolname}' group with 'adhoc' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, scheduling_policy, environment, parent) VALUES ('bi-${toolname}', '10%', 10, 100, 10, 'weighted_fair', 'test_environment', 3);
+
+    -- create 'pipeline' group with 'global' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, jmx_export, environment, parent) VALUES ('pipeline', '80%', 45, 100, 1, true, 'test_environment', 1);
+
+    -- create a root group 'global' with NULL parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_policy, jmx_export, environment) VALUES ('global', '80%', 100, 1000, 'weighted', true, 'test_environment');
+
+    -- get ID of 'global' group
+    SELECT resource_group_id FROM resource_groups WHERE name = 'global';  -- 1
+    -- create two new groups with 'global' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, environment, parent) VALUES ('data_definition', '10%', 5, 100, 1, 'test_environment', 1);
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, environment, parent) VALUES ('adhoc', '10%', 50, 1, 10, 'test_environment', 1);
+
+    -- get ID of 'adhoc' group
+    SELECT resource_group_id FROM resource_groups WHERE name = 'adhoc';   -- 3
+
+    -- create 'other' group with 'adhoc' as parent
+    INSERT INTO resource_groups (name, soft_memory_limit, hard_concurrency_limit, max_queued, scheduling_weight, scheduling_policy, environment, parent) VALUES ('other', '10%', 2, 1, 10, 'weighted_fair', 'test_environment', 3);
+
+--In below examples, three selectors are created for user1, user2, and user3 respectively.
```

Comment:
I suggest you look at the formatting of the content at the bottom of the page beginning at this line. Is this how you want it to look, with dashes at the beginning of the sentences and the indentation creating the vertical bar? 

Also, consider formatting named items such as resource_group_id, resource_groups, and user_regex as code with `` before and after. 

<img width="836" alt="Screenshot 2023-09-07 at 11 47 40 AM" src="https://github.com/prestodb/presto/assets/7013443/f59e51b6-1a4c-43ab-9547-83c976e96060">